setup.py

```diff
@@ -6,7 +6,8 @@
 Packages:   diffpy.nmf_mapping
 """
 
-from setuptools import find_packages, setup
+import setuptools
```

Comment:
maybe handle it the same way, either just import setuptools, or import the modeles we want and refer to them directly?